src/callbacks_step/time_series.jl

```diff
@@ -24,7 +24,8 @@ The real data type `RealT` and data type for solution variables `uEltype` defaul
 types used in the solver and the cache.
 
 !!! warning "Experimental implementation"
-    This is an experimental feature and may change in future releases.
+    This is an experimental feature and may change in future releases. Currently this callback
+    is only implemented for [`TreeMesh`](@ref) in 2D and [`UnstructuredMesh2D`](@ref).
```

Comment:
I think with all your work, we can remove the experimental warning, can't we? We can still mention that it's implemented only for the `TreeMesh` and the `UnstructuredMesh2D`.

Reply:
We can discuss this at the Trixi meeting. I am okay if this callback becomes non-experimental, but we can get everyone's opinion on the matter.

Reply:
I removed the "Experimental implementation" warning from the docstring.